### PR TITLE
ci: open release pull request automatically

### PR DESCRIPTION
**Pull request by rmoles**
Back porting this update to the release.sh script lacework/go-sdk@f227182
Signed-off-by: Ross Moles <ross.moles@lacework.net>

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -259,8 +273,8 @@ bump_version() {
   fi
 
   log "commiting and pushing the version bump to github"
-  git config --global user.email "$git_email"
-  git config --global user.name "$git_user"
+  git config --global user.email $git_email
+  git config --global user.name $git_user
```
Comment (afiune) [inline scripts/release.sh:277]: 🤔 Not even the user needs the double quoting here¿? `git_user="Lacework Inc."`

**Review by afiune — APPROVED** [review]
If the `git_user` variable will be expanded without quotes, then Im 👍🏽

File: scripts/release.sh (hunk 1 above)
Comment (rmoles) [reply scripts/release.sh:277]: This was copied from the services account repo. Shall I raise a PR to update there also?

Comment (afiune) [reply scripts/release.sh:277]: 😂 I don't even know what day is today. Please, disregard this comment.

**Pull request merged**